xchange-binance/src/main/java/org/knowm/xchange/binance/dto/trade/margin/MarginSideEffectType.java

```diff
@@ -0,0 +1,19 @@
+package org.knowm.xchange.binance.dto.trade.margin;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import org.knowm.xchange.dto.Order;
+
+public enum MarginSideEffectType implements Order.IOrderFlags {
+  NO_SIDE_EFFECT,
+  MARGIN_BUY,
+  AUTO_REPAY;
+
+  @JsonCreator
+  public static MarginSideEffectType getSideEffectType(String s) {
```

Comment:
this method seams pointless as jacksons standard error handling will do a much at reporting an unknown value

Reply:
Agree, but this is how it's implemented for other Binance enums. So I just copied the same approach